Dockerfile

```
@@ -34,10 +34,12 @@ cp Video_Codec_SDK_13.0.19/Interface/* /usr/local/cuda/include && \
 cmake .. -DUSE_CUDA=ON -DCMAKE_BUILD_TYPE=Release && \
 make -j 4 && \
 cd ../python && python3 setup.py install; fi
-
 RUN if [[ ${TARGETPLATFORM} == 'linux/arm64' ]]; then apt remove -y python3-blinker; fi
+
 # Install the dependencies from requirements-docker.txt
 COPY ./requirements-docker.txt /requirements.txt
+ARG NATTEN_CUDA_ARCH="8.0;8.6;8.9;9.0;10.0;10.3"
```

```suggestion
ARG NATTEN_CUDA_ARCH="8.0;8.6;8.9;9.0;10.0;10.3;12.0"
```

Can we add SM120 (Blackwell RTX) as well?

---

done